spec/services/state_service_spec.rb

```diff
@@ -139,4 +139,28 @@
       end
     end
   end
+
+  describe "#accessioned?" do
+    subject(:accessioned?) { service.accessioned? }
+
+    context "if the accessioned lifecycle exists" do
+      before do
+        allow(workflow_client).to receive(:lifecycle).with(druid:, milestone_name: "accessioned").and_return("2022-04-20 21:55:25 +0000")
+      end
+
+      it "returns true" do
+        expect(subject).to be true
```

Comment:
We could also consider ditching line 144, since we don't appear to be referring to the `accessioned?` variable, and changing this line to use a predicate matcher:


```suggestion
        expect(service).to be_accessioned
```